actions/project.go

```diff
@@ -59,12 +60,26 @@ func DownloadTemplate(c *cli.Context) {
 func ValidateProject(c *cli.Context) {
 	projectPath := c.Args().Get(0)
 	utils.CheckProjectPath(projectPath)
+	validationStatus := "success"
+	// result could be ProjectType or string, so define as an interface
+	var validationResult interface{}
+	language, buildType, isAppsody := utils.DetermineProjectInfo(projectPath)
+	validationResult = ProjectType{
+		Language: language,
+		BuildType: buildType,
```

Comment:
`buildType` needs to be `projectType` in the JSON. See: https://github.com/eclipse/codewind-installer/pull/97/files (which is awaiting review).

Reply:
I've pulled that change in by re-basing